recipes/nanort/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,27 @@
+#include <cstdlib>
+#include <iostream>
+#include <array>
+
+#include <nanort/nanort.h>
```

Comment:
checking the examples: https://github.com/lighttransport/nanort/blob/release/examples/bidir_path_tracer/main.cc I can see that the typical include is directly like: `#include "nanort.h"`